translations/hi.json

```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "en": "Hindi"
```

Comment:
Please change "en" to "hi". Language names should also be written in the Hindi language.